src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test.Common/Microsoft.AspNetCore.Razor.LanguageServer.Test.Common.csproj

```diff
@@ -12,6 +12,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.EditorFeatures.Common" Version="$(Tooling_MicrosoftCodeAnalysisEditorFeaturesCommonPackageVersion)" />
```

Comment:
Roslyn's hover handler lives in the EditorFeatures dll so this new reference was needed